dataland-backend/src/main/kotlin/org/dataland/datalandbackend/services/DataMetaInformationManager.kt

```diff
@@ -91,17 +92,23 @@ class DataMetaInformationManager(
     }
 
     /**
-     * Method to make the data manager search for meta info
-     * @param companyId if not empty, it filters the requested meta info to a specific company
-     * @param dataType if not empty, it filters the requested meta info to a specific data type
-     * @param reportingPeriod if not empty, it filters the requested meta info to a specific reporting period
-     * @param showOnlyActive if true, it will only return datasets marked "active"
-     * @return a list of meta info about data depending on the filters
+     * Method to make delete the data meta information for a given dataId
```

Comment:
Doc sounds strange

Reply:
Will be pushed in 2 minutes

Reply:
thank you for correcting the whole doc for the delete function!